test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2310,6 +2310,93 @@ def forward(self, x, y):
         y = torch.rand(100)
         self.run_test(PowModule3(), (x, y))
 
+    # the standardOps(Add\Sub\Mul\Div\Gemm\Pow\Mod) with low precision include unit8 will be failed in ORT
+    # add to(dtype=torch.long) to avoid ORT output type does not match expected type
+    def test_standardOps_with_low_precision(self):
```

Comment:
nit: suggest rename `standardOps` as `arithmeticOps`

Reply:
Does Gemm\Pow\Mod no need any more? It's also reasonable for it.